### DISCUSSION TITLE
肉鸽非商店的藏品选择逻辑是不是和shoppings.json无关

### DISCUSSION THREAD

**Discussion by ChenParsifal** [MaaAssistantArknights/MaaAssistantArknights#9325] [Q&A] [answered]
例如打完第三层boss选藏品的时候，猎人的洞察明明优先级很高，但是却不会被优先选择。
是因为有什么特殊的理由和判断吗。
asst.log里面好像只有板子会被ocr识别，是因为没有识别所以盲选的吗。

**Comment by ABA2396** [comment, marked as answer]
选左边

**Reply by ChenParsifal** [reply]
谢谢。
观察了最近的两把，一把是郁金香和蒸汽甲胄，确实选了左边。
但是再之前一局可能是因为有四叶草，从左到右是猎人的洞察，教母的信物，损坏的左轮弹巢。
结果最后选了最右边的左轮。
